app/assets/src/components/Samples.jsx

```diff
@@ -103,7 +104,9 @@ class Samples extends React.Component {
       selectedProjectId: this.fetchParams("project_id") || null,
       filterParams: this.fetchParams("filter") || "",
       searchParams: this.fetchParams("search") || "",
-      sampleIdsParams: this.fetchParams("ids") || [],
+      sampleIdsParams: this.fetchParams("ids")
```

Comment:
Can you move this logic to `fecthParams`.
fetchParams should, for certain fields, split into an array.

Reply:
good idea